configs/eslint-config-base/package.json

```diff
@@ -17,9 +17,11 @@
     "eslint-config-prettier": "^8.5.0",
     "eslint-plugin-eslint-comments": "^3.2.0",
     "eslint-plugin-import": "^2.26.0",
-    "eslint-plugin-jest": "^27.1.5",
+    "eslint-plugin-jest": "^25.3.0",
     "eslint-plugin-prettier": "^4.2.1",
-    "eslint-plugin-promise": "^6.1.1"
+    "eslint-plugin-promise": "^6.1.1",
+    "eslint-utils": "^3.0.0",
+    "regexpp": "^3.2.0"
```

Comment:
why did you add these dependencies in the base config?

Reply:
Done